pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.networknt</groupId>
     <artifactId>json-schema-validator</artifactId>
-    <version>1.0.67</version>
+    <version>1.0.68-SNAPSHOT</version>
```

Comment:
I don't know how @stevehu did release management before but I'd leave it at 1.0.67 and let him bump the version once the next release is planned.

Reply:
Fine - I just did this for my personal safety in order not to overwrite the official 1.0.67 in both my Maven repo and/or our Artifactory by a development build from my machine.

Reply:
Don't worry about this. I will have to update this version before the release anyway. Let's keep the change.